Sources/ManageBoards.php

```diff
@@ -697,7 +697,7 @@ function EditBoard2()
 		$boardOptions['redirect'] = !empty($_POST['redirect_enable']) && isset($_POST['redirect_address']) && trim($_POST['redirect_address']) != '' ? trim($_POST['redirect_address']) : '';
 
 		// Profiles...
-		$boardOptions['profile'] = $_POST['profile'];
+		$boardOptions['profile'] = $_POST['profile'] == -1 ? 1 : $_POST['profile'];
```

Comment:
I have a question. Is there a reason why you chose `==` instead of `===`?

Reply:
It was like that on the row below

Reply:
Enough for me, thanks!